run_create_appstream_stack_fleet.py

```diff
@@ -222,11 +238,14 @@ def get_latest_image(image_region):
         fleet_name = settings['FLEET_NAME']
         region = settings['AWS_DEFAULT_REGION']
         stack_name = settings['NAME']
+        canonical_id = settings['CANONICAL_ID']
+        fleet_arn = f'arn:aws:iam::{canonical_id}:role/AmazonAppStreamS3Permission'
```

Comment:
@GoldieKMG // arn 관련 변수는 무엇에 대한 arn인지 분명하게 써주는게 좋습니다. 그리고 ARN 주소는 AWSCli 에서 role 생성할때 가져온걸 그대로 박는게 좋습니다. 문자열로 임의 생성하면 문제가 될수 있습니다.
```suggestion
        fleet_role_arn = f'arn:aws:iam::{canonical_id}:role/AmazonAppStreamS3Permission'
```